### PR TITLE
Add benchmarks

### PR DESCRIPTION
**Pull request by sgugger**
This PR adds a benchmark script to the repository showing the speed of the big model inference using Accelerate on several big models. The goal is to compare it to other libraries to show the strengths and weaknesses of each.

I have just run the benchmark on my machine for now. Sharing this script will make it easier to tests other kinds of setups!

### CODE REVIEW

File: benchmarks/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Big model inference benchmarks
+
+Running inference with Accelerate on big models.
+
+## Setup
+
+These benchmarks use the `Transformers` library:
```
Comment (muellerzr) [inline benchmarks/README.md:7]:
```suggestion
These benchmarks use the `transformers` library:
```

File: benchmarks/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+```bash
+pip install transformers
+```
```
Comment (muellerzr) [inline benchmarks/README.md:11]: Do they not also need `measures_util`?

Maybe add a `pip install accelerate[bench]`?

**Review by muellerzr — APPROVED** [review]
Looks great! Just left a few notes

File: benchmarks/README.md (hunk 2 above)
Comment (sgugger) [reply benchmarks/README.md:11]: Forgot one file :grimacing:

**Pull request merged**